tutorials/multi-tenant-tutorial.rst

```diff
@@ -0,0 +1,166 @@
+                                                                                             
+Next, you can create primary key indexes on each of the tables just like you would do in PostgreSQL
+    
+::
+                                                                                         
+    ALTER TABLE companies ADD PRIMARY KEY (id);                                                  
+    ALTER TABLE campaigns ADD PRIMARY KEY (id, company_id);                                      
+    ALTER TABLE ads ADD PRIMARY KEY (id, company_id);
+
+Distributing tables and loading data
+-------------------------------------
+
+We will now go ahead and tell Citus to distribute these tables across the different nodes we have in the cluster. To do so,
+you can run :code:`create_distributed_table` and specify the table you want to shard and the column you want to shard on.
+In this case, we will shard all the tables on the :code:`company_id`.                             
+                                                                                          
+::
+    
+    SELECT create_distributed_table('companies', 'id');                                       
+    SELECT create_distributed_table('campaigns', 'company_id');                               
+    SELECT create_distributed_table('ads', 'company_id');                                     
+                                                                                          
+Sharding all tables on the company identifier allows Citus to `colocate <colocation>`_ all the tables 
+and allow for features like primary keys, foreign keys and complex joins across your cluster.
+You can learn more about the benefits of this approach `here <https://www.citusdata.com/blog/2016/10/03/designing-your-saas-database-for-high-scalability/>`_.
+                                                                                          
+We could now go ahead and load data into these tables. You can download sample files for all the tables here:
+companies, campaigns and ads. Once you have these files, you can go ahead and load them using the standard PostgreSQL :code:`\COPY` command.
+
+::
+                                                                                          
+    \copy companies from 'companies.csv' WITH CSV;                                                     
+    \copy campaigns from 'campaigns.csv' WITH CSV;                                                     
+    \copy ads from 'ads.csv' WITH CSV;
+
+Running queries and modifications
+---------------------------------
+
+Now that the data is loaded, let's go ahead and run some queries. You could start with a simple count
+query to see how many companies we have data for.              
+                                                                                          
+::
+
+    SELECT count(*) from companies;                                                           
```

Comment:
What do you think about swapping one of the following three ``SELECT`` statements with ``INSERT``s?

That way, our tutorial shifts some emphasis from the OLAP to the OLTP side?